### PR TITLE
feature(core): redesign from and to (4)

### PR DESCRIPTION
**Pull request by pierregee**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- This PR is to redesign the from and to. It includes the validation for insufficient balance

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
Out of scope:
- UTXO conversion warning design 
- Token Selection screen
- When switching from instant <-> future swap, all the values reset. This will be updated in a separate PR
- Transaction fees (not removed yet)

image
image
image

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx [hunk 1]
```diff
@@ -89,6 +82,12 @@ import {
   ButtonGroupTabKey,
   SwapButtonGroup,
 } from "./components/SwapButtonGroup";
+import { TokenDropdownButton } from "./components/TokenDropdownButton";
+import { ActiveUSDValueV2 } from "../../Loans/VaultDetail/components/ActiveUSDValueV2";
+import {
+  WantFutureSwapRow,
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx:88]: Why is this Want? Let's rename it to From or To

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx [hunk 2]
```diff
@@ -601,6 +610,17 @@ export function CompositeSwapScreenV2({ route }: Props): JSX.Element {
     }
   };
 
+  const getAmountInUSDValue = useCallback(
+    (token: OwnedTokenState | TokenState | undefined, tokenAmount: any) => {
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx:614]: Change type of tokenAmount

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx (hunk 1 above)
Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx]: Updated! was torned if should name it based on the UI label or its functionality 
Updated to ```InstantSwapRowTo and FutureSwapRowTo```

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx (hunk 2 above)
Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx:611]: Updated!

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**